graphql/schema.graphql

```diff
@@ -202,6 +202,9 @@ type Query {
     """ The page size """
     pageSize: Int = 15
   ) : [List]
+
+  """ Get last order of logged in user """
+  lastUserOrder: Order @cacheControl(scope: PRIVATE)
```

Comment:
I think the name could be

```
usersLastOrder
```

What do you think?

Reply:
changed to `userLastOrder`